### ISSUE TITLE
Rewrite in Rust

### ISSUE THREAD

**Issue by overlisted** [vladimirpichugin/HypeBot#3]
Using Python in production code is a programming sin and increases your chance of being placed in the programmer's hell after your death if don't immediately take action. A modern ( 🚀) and type-safe ( 🚀) language with zero-cost abstractions ( 🚀) like [Rust](https://rust-lang.org) ( 🚀) would greatly improve the performance of your project and decrease your chance of spending your afterlife in the programmer's hell. If I were in your place, I would definitely choose Rust ( 🚀) over Python, given all the benefits of Rust ( 🚀) compared to Python.

Here's a repository that you can use as a template for your bot's code, if you decide to use Rust ( 🚀): https://github.com/mTvare6/hello-world.rs

**Comment by zemlyak** [comment]
I strongly agree with this issue. Rust can be a very great way to increase the performance of an application to C/C++ level without having to deal with numerous issues of those languages such as buffer overflows and other errors that are being caused by memory bugs.
